src/components/reportRepair/address.js

```diff
@@ -57,7 +57,7 @@ const Address = ({handleChange, values, addresses}) => {
           <Link as={RouterLink} to="not-eligible">I can&apos;t find my address on this list</Link>
```

Comment:
The title/headline of this page is no longer "Where is the repair located?" and should be changed to "Select an address".